admin-dev/themes/new-theme/js/app/utils/serp/index.js

```diff
@@ -107,10 +107,15 @@ class SerpApp {
   }
 
   setUrl(rewrite) {
+    // We replace two placeholders because there was a typo in the initial one
```

Comment:
```suggestion
    // We replace two placeholders because there was a typo in the initial one ('friendy' instead of 'friendly')
```
Just to make it easier to read 😉